Package.swift

```diff
@@ -0,0 +1,44 @@
+// swift-tools-version:5.3
+import PackageDescription
+
+let package = Package(
+    name: "NYTPhotoViewer",
+    platforms: [
+        .iOS(.v9)
+    ],
+    products: [
+        .library(
+            name: "NYTPhotoViewer",
+            targets: ["NYTPhotoViewer"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/pinterest/PINRemoteImage.git", from: "3.0.1")
+    ],
+    targets: [
+        .target(
+            name: "NYTPhotoViewer",
+            dependencies: ["PINRemoteImage"],
+            path: "NYTPhotoViewer",
+            exclude: [
+                "Info.plist"
+            ],
+            resources: [
+                .process("Resources/NYTPhotoViewerCloseButtonX.png"),
```

Comment:
let's put these in an asset catalog if possible

Reply:
i'll give it a shot, but i'm not hopeful.

Reply:
In 43913b3, I replaced the `Resources` directory with an .xcassets catalog. Give it a shot!

Reply:
Nice! That works well (both in that it cleans the package manifest up a bit and in that I tested it end-to-end to confirm the assets carry through to runtime).